pkg/kubecost/audit.go

```diff
@@ -38,8 +38,8 @@ func ToAuditType(check string) AuditType {
 		return AuditAssetTotalStore
 	case string(AuditAssetAggStore):
 		return AuditAssetAggStore
-	//case string(AuditClusterEquality):
-	//	return AuditClusterEquality
+	case string(AuditClusterEquality):
+		return AuditClusterEquality
```

Comment:
It's nice to have this to rely on -- I'm assuming that uncommenting this means that it passes now?

Reply:
It passes our tests, and on my cluster, given everything that I have seen in the Costmodel in the last couple weeks I don't think it will pass in all situations but I think it is a good audit have in that it lets us know our short comings on this invariant.